packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -718,18 +718,6 @@ class InspectorController extends DisposableController
     // a node is selected, its properties will also be selected as by
     // convention properties are the first children of a node and properties
     // typically do not have children and are never expanded by default.
-    for (InspectorTreeNode child in node.children) {
```

Comment:
why was this logic removed? If the problem is it does bad things with too many properties, we can tweak the logic to only require some small number of children.

Reply:
Scrolling to the top left corner of the screen will be jarring for cases like going through a list of search results.
For example image stepping through search results and you get one that is slightly off the screen but now you scroll ALL the way up till it is at the top of the screen.
I would try to maintain the previous functionality which is pretty much industry standard for how you should scroll to bring some new content to be visible on the screen. Instead I would special case so we abandon making the parent visible when it is not possible as the list of properties is larger than the height of the screen.

Reply:
We chatted about this during a 1:1 and decided together that scrolling to the center of the screen is a happy medium we can agree on.